src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -2,11 +2,29 @@
   
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+
+    <!-- To Publish single-file exes, run: 
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r win-x64
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r linux-x64
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r osx-x64
+
+      For settings which we want only for these builds, we use [Condition="'$(RuntimeIdentifier)' != ''"] to identify them
+    -->
+
+    <!-- Keep default targets to allow VS2017 use; our build pool is Hosted VS2017 only. -->
```

Comment:
>our build pool is Hosted VS2017 only [](start = 51, length = 36)

Since this is a public repo, I wouldn't mention "our" build pool. Maybe just "Build for frameworks prior to .NET Core 3.0 to enable building on VS2017."

Reply:
It turns out the Hosted VS2017 build agent pool is public; I've updated the comment to be clearer.